### DISCUSSION TITLE
cmake cuda arch issues

### DISCUSSION THREAD

**Discussion by leezu** [apache/incubator-mxnet#9516] [Q&A]
@cjolivier01 there still seem to be some issues with cuda arch and cmake.

Ref: #6204 #6799

In particular I experience an issue similar to #6204: With cmake: `Check failed: err == cudaSuccess (7 vs. 0) Name: MapRedKeepLowestKernel ErrStr:too many resources requested for launch` whereas the ordinary Makefile build works. In both cases I target `-gencode arch=compute_61,code=sm_61` for GTX 1080.

**Comment by cjolivier01** [comment]
When the CMake config occurs, you see that it is selecting sm61 in the output?
ie my work box says:
--  CUDA_ARCH_LIST: Auto
-- Autodetected CUDA architecture(s): 5.0
-- cuda arch bin: 50
-- cuda arch ptx:

**Comment by vdantu** [comment]
@sandeep-krishnamurthy: Label "CUDA", "Question"
@leezu : If you are still seeing this issue, could you give further information to @cjolivier01?

**Comment by octi-biris** [comment]
Bug still present using the cmake build on a TITAN V card. the architectures that the binary targetted is sm_70. The makefile build works but all known architectures are targetted.
